### DISCUSSION TITLE
Windows 10 issue?

### DISCUSSION THREAD

**Discussion by Hendo104** [SnosMe/awakened-poe-trade#861] [Q&A]
Using 3 monitors and windows 10,  when trying to price check it just makes the other screens go black.  Reinstalled multiple times, tried the --disable-gpu command line in properties and nothing fixes it.  What else can I do? Is this a Nvidia settings issue?

**Comment by SnosMe** [comment]
I have no idea how it can affect other screens. I have a second monitor too.
